lib/builderator/config/file.rb

```diff
@@ -0,0 +1,210 @@
+require 'time'
+
+require_relative './attributes'
+require_relative '../util'
+
+module Builderator
+  module Config
+    ##
+    # DSL Loader for a configuration file
+    ##
+    class File < Attributes
+      class << self
+        ## DSL Loaders
+        def from_file(source, **options)
```

Comment:
`from_file` is a fairly common convention in Ruby DSLs, including Chef and Berkshelf